assets/js/components/Beneficiaries.vue

```diff
@@ -332,6 +390,45 @@ export default Vue.extend({
         .transition(t_)
         .call(this.renderItems);
 
+
+      // tooltip
+      let tip = d3.tip()
+        .attr('class', 'd3-tip benef')
+        .html(function(d){
+         return "<div class='title-container'><img src=/assets/imgs/"
```

Comment:
a) there is specific css ready-made for country flags.
b) you have the country in `d`. d3.select(this).select('use').attr('href').substring(1) is ... not right.